bash/bash_vcs.sh

```diff
@@ -84,7 +84,15 @@ __prompt_command() {
 	}
 
 	bzr_dir() {
-		base_dir=$(bzr root 2>/dev/null) || return 1
+		# bzr root is really slow, so we simulate it
+        	# with a bash script I modified frome here:
+        	# http://unix.stackexchange.com/questions/6463/find-searching-in-parent-directories-instead-of-subdirectories
+		bzr_root=$(pwd -P 2>/dev/null || command pwd)
+        	while [ ! -e "$bzr_root/.bzr" ]; do
+            		bzr_root=${bzr_root%/*}
+            		if [ "$bzr_root" = "" ]; then break; fi
+        	done
+		base_dir=$bzr_root || return 1
```

Comment:
This line doesn't quite work yet. I haven't had time to fix it.
At the moment, this is a regression, because it shows (bzr) even when you are not a a bzr directory.

This is because $bzr_root can never return a non zero exit status, because it is not a function.

I don't know quite enough bash to fix this, and don't have time to learn at the moment.